client/app/layouts/application.html

```diff
@@ -28,6 +28,28 @@
             <span class="fa" ng-class="{'fa-bell': vm.newNotifications, 'fa-bell-o': !vm.newNotifications}"></span>
           </a>
         </li>
+        <li class="dropdown">
+          <a class="dropdown-toggle nav-item-iconic" id="aboutModal" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+            <span class="fa pficon-help"></span>
+            <span class="caret"></span>
+          </a>
+          <ul class ="dropdown-menu" aria-labelledby="aboutModal">
+            <li>
+              <a href="/support/index?support_tab=about">
+                <span translate>
```

Comment:
I don't think this is how we want to do translates anymore.  See https://github.com/ManageIQ/manageiq-ui-self_service/pull/182 for good examples.